sdk/iot/common/inc/az_iot_common_internal.h

```diff
@@ -32,6 +32,16 @@
  */
 AZ_NODISCARD int32_t _az_iot_u32toa_size(uint32_t number);
 
+/**
+ * @brief Copies the url-encoded content of `source` span into `destination`, returning the free remaining of `destination`.
+ *
+ * @param[in] destination The span where the url-encoded content is copied to.
```

Comment:
@param[in] destination The span where the source is url-encoded to.

Reply:
Fixed.